__tests__/app.test.js

```diff
@@ -85,6 +85,54 @@ afterAll(() => {
             })
         })
     })
+    test("status: 404 when passed an id that could not be found", () => {
+        return request(app)
+        .patch("/api/articles/1000")
+        .send({ inc_votes: 150 })
+        .expect(404)
+        .then(({body}) => {
+            expect(body.msg).toEqual("Article not found")
+        })
+    })
+    test("status: 400 when passed an invalid id", () => {
+        return request(app)
+        .patch("/api/articles/bannana")
+        .send({inc_votes: 150})
+        .expect(400)
+        .then(({body}) => {
+            expect(body.msg).toEqual("Invalid article_id")
+        })
+    })
+    test("status: 400 when passed an invalid inc_votes value", () => {
+        return request(app)
+        .patch("/api/articles/4")
+        .send({inc_votes: "bannana"})
+        .expect(400)
+        .then(({body}) => {
+            expect(body.msg).toEqual("Invalid inc_votes value")
+        })
```

Comment:
you could add another error here to deal with no inc_votes key or an empty object. you could either send this back as a 400 bad request or 200 and just respond with the same object!